### DISCUSSION TITLE
UI Bug on "SSH and GPG keys" Page

### DISCUSSION THREAD

**Discussion by nekonado** [community/community#126808] [Other Feature Feedback, Questions, & Ideas] [answered]
I have discovered what appears to be a UI bug on the "SSH and GPG keys" page.

- **Page URL:** [https://github.com/settings/keys](https://github.com/settings/keys)
- **Issue:** In the GPG keys section, the text on the Delete button is overflowing.

Below is a screenshot from my environment showing the issue:

<img alt="" src="https://github.com/community/community/assets/94698042/ca4edf86-80bd-4b10-8f82-c9c3d4288edc">

Please let me know if you need any further information.

**Comment by mickhence** [comment]
"We've identified a UI bug on the 'SSH and GPG keys' page and our team is currently working on fixing it. Thank you for bringing it to our attention!"

**Comment by ebndev** [comment, marked as answer]
Hi @nekonado, thanks for flagging this!

I will keep a log of this and note if other users have this issue this as well. The GitHub product team will evaluate the feedback but will not be able to respond to every submission. You can keep an eye on our [public roadmap](https://github.com/github/roadmap/projects/1) to see the kinds of work we are focusing on for the coming year. Smaller fixes won’t be listed, but are often included when work is happening adjacent to it.

Thank you for contributing and continuing to support other users in the GitHub Community Discussions.

**Comment by nekonado** [comment]
Hi, @ebndev.

I have confirmed that this issue has been resolved in my environment, either because it was a temporary incident or thanks to the efforts of The GitHub product team.

Thank you as always for the excellent service!

**Reply by ebndev** [reply]
@nekonado, that's great news!

Thanks for taking the time to respond and for being a part of the GitHub Community 😄
